leakcanary-android-core/src/main/java/leakcanary/internal/SquigglySpan.kt

```diff
@@ -15,20 +15,22 @@
  */
 package leakcanary.internal
 
+import android.content.Context
 import android.content.res.Resources
 import android.graphics.Canvas
 import android.graphics.Paint
 import android.graphics.Path
 import android.text.SpannableStringBuilder
 import android.text.style.ReplacementSpan
 import android.text.style.UnderlineSpan
+import androidx.core.content.ContextCompat
 import com.squareup.leakcanary.core.R
 
 /**
  * Inspired from https://github.com/flavienlaurent/spans and
  * https://github.com/andyxialm/WavyLineView
  */
-internal class SquigglySpan(resources: Resources) : ReplacementSpan() {
+internal class SquigglySpan(context: Context, resources: Resources) : ReplacementSpan() {
```

Comment:
Same as elsewhere. Also if we go that route  we should remove Resources. Not a big fan of having context everywhere though

Reply:
Ah yeah that's a good idea; forgot we could just pull the resources from the view context I was using here 👍